docs/pages/enroll-resources/server-access/rbac.mdx

```diff
@@ -135,8 +135,18 @@ spec:
     create_host_user_mode: keep
     # forward_agent controls whether SSH agent forwarding is allowed
     forward_agent: true
-    # port_forwarding controls whether TCP port forwarding is allowed for SSH
-    port_forwarding: true
+    # ssh_port_forwarding controls which TCP port forwarding modes are allowed over SSH. This replaces
+    # the now deprecated port_forwarding field which did not differentiate between remote and local
```

Comment:
```suggestion
    # the deprecated port_forwarding field, which did not differentiate between remote and local
```
- removing "now", since the docs are assumed to be up to date as of the present
- adding a comma before "which"